x-pack/plugins/infra/public/common/visualizations/lens/utils.ts

```diff
@@ -12,8 +12,9 @@ import {
 import type { DataView, DataViewSpec } from '@kbn/data-views-plugin/public';
 import type { SavedObjectReference } from '@kbn/core-saved-objects-common';
 
-export const DEFAULT_LAYER_ID = 'layer1';
+export const DEFAULT_LAYER_ID = 'layer';
```

Comment:
Q: Why is this changed?

Reply:
This is just the base id used to build the actual layer id. There can be multiple layers, so in the end, the layer ids will look like layer_0, layer_1, layer_3..., provided that there are multiple layers.

Reply:
Ok, I see, thanks for explaining that